assets/pages/new-contributions-landing/contributionsLandingInit.js

```diff
@@ -71,15 +75,29 @@ function getInitialContributionType(abParticipations: Participations): Contribut
   );
 }
 
-function initialiseStripeCheckout(onPaymentAuthorisation, contributionType, currencyId, isTestUser, dispatch) {
+const stripeAccountForContributionType: {[ContributionType]: StripeAccount } = {
+  ONE_OFF: 'ONE_OFF',
+  MONTHLY: 'REGULAR',
+  ANNUAL: 'REGULAR',
+};
+
+
+function initialiseStripeCheckout(
+  onPaymentAuthorisation: (paymentAuthorisation: PaymentAuthorisation) => void,
+  contributionType: ContributionType,
+  currencyId: IsoCurrency,
+  isTestUser: boolean,
+  dispatch: Function,
+) {
   const library: ThirdPartyPaymentLibrary =
-    setupStripeCheckout(onPaymentAuthorisation, contributionType, currencyId, isTestUser);
+    setupStripeCheckout(
+      onPaymentAuthorisation,
+      stripeAccountForContributionType[contributionType],
+      currencyId, isTestUser,
```

Comment:
is this trailing comma required?

Reply:
It is, although `isTestUser` should be on a new line.

Reply:
I'll fix now!